### ISSUE TITLE
Cannot access CK bad class file

### ISSUE THREAD

**Issue by M12Shehab** [mauricioaniche/ck#30]
Hi all

Firstly, many thanks for this powerful tool that helps us in our researches.
I tried to add Gradle dependency to my Java implementation to use this library. I got this issue when I tried to build the project.

I tried many solutions and I gave up. Can you please help me to solve it?

I used intellij IDM, Java 1.8 and javac 1.8.0_201.

Thanks,

**cannot access CK bad class file: C:\Users\GIGABYTE\.gradle\caches\modules-2\files-2.1\com.github.mauricioaniche\ck\0.4.3\3363cb42f4417452fe749730dc813f68c658a840\ck-0.4.3.jar(com/github/mauricioaniche/ck/CK.class) class file has wrong version 55.0, should be 52.0 Please remove or make sure it appears in the correct subdirectory of the classpath.**

**Comment by mauricioaniche** [comment]
Hi, @M12Shehab. You need to use Java 11 to compile CK. I'm really thinking of back porting it, as I have other people who only have access to Java 8. I'll give it a try and post some instructions soon.

**Comment by mauricioaniche** [comment]
Done, @M12Shehab! Use CK version 0.4.4 that I just deployed on Maven. This one should be compatible with JDK 8.

**Comment by M12Shehab** [comment]
Hi @mauricioaniche 

Thanks for your response. I tried to build it separately and it works perfectly.

I found the solution and I think it will help others.

On the right corner of your in IntelliJ IDM, click edit config and change the JRE to 13 or higher Java version than Java 8
![Untitled](https://user-images.githubusercontent.com/18244361/69578405-6ca19680-0f9e-11ea-8fe2-8edb99dda6e9.png)

Thanks :)
